### ISSUE TITLE
Force language of responses

### ISSUE THREAD

**Issue by ArbiterOfBuffoonery** [a85/POSTMan-Chrome-Extension#1653]
I am working with a service that does not correctly set the Content-Type of responses. This results in postman interpreting the XML response as HTML and changing the response from 

`<someTag date="2015-12-01">`
to
`<someTag date="2015&#x2D;12&#x2D;01">`

Is there a way to force the response to be a specific format? I have tried setting the response to XML from the dropdown above the response content, but it has no effect. 
![screenshot](https://cloud.githubusercontent.com/assets/13684150/12251264/93b2e9da-b89b-11e5-8e11-25333c18b48d.png)

Currently I can set the language detection to Auto or JSON. Would it be possible to add XML so that the response would not be adjusted?

Thanks

**Comment by abhijitkane** [comment]
@paradoX-D Can you include the output of the raw tab here?

**Comment by ArbiterOfBuffoonery** [comment]
@abhijitkane Below is the contents of the raw response. Note that I have verified that this is not what the server responds with (by viewing the output right before it is sent). Please let me know if you need anything else.

```
<RESPONSE value1="123" value2="8" value3="1234" value4="aValue" value5="bValue" value6="cValue"><NODE value7="123" date="2016&#x2D;01&#x2D;31"></NODE></RESPONSE>
```

**Comment by abhijitkane** [comment]
@paradoX-D That's odd, Postman performs no manipulation for the raw body. Can you check the Network tab of the DevTools window (http://blog.getpostman.com/2015/06/13/debugging-postman-requests/)? This will show the exact response that Postman receives. (You'll need to check the Response tab of your request)

**Comment by ArbiterOfBuffoonery** [comment]
@abhijitkane I apologize. I have wasted your time :-(

After reviewing the raw response using the DevTools, I realized you were correct and that there was no way that Postman was changing the response. There is a library that I was using that was doing this. When that library was updated this behavior was introduced. The update to the library was not obvious and also happened to be right after I updated Postman. All the testing I was doing against postman was therefore based on an invalid assumption so my test results so were not reflective of reality.

Sorry again to have not been more thorough in my testing. Thanks for looking into this.
